homeassistant/components/bsblan/diagnostics.py

```diff
@@ -7,17 +7,23 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 
-from . import HomeAssistantBSBLANData
+from . import BSBLanData
 from .const import DOMAIN
 
 
 async def async_get_config_entry_diagnostics(
     hass: HomeAssistant, entry: ConfigEntry
 ) -> dict[str, Any]:
     """Return diagnostics for a config entry."""
-    data: HomeAssistantBSBLANData = hass.data[DOMAIN][entry.entry_id]
+    data: BSBLanData = hass.data[DOMAIN][entry.entry_id]
+
     return {
         "info": data.info.to_dict(),
         "device": data.device.to_dict(),
-        "state": data.coordinator.data.to_dict(),
+        "coordinator_data": {
+            "state": data.coordinator.data.state.to_dict()
+            if data.coordinator.data
+            else {},
+        },
+        "static": data.static.to_dict(),
```

Comment:
Ideally we change this in a follow up btw, let's do that as its not redacted yet

Reply:
So we leave this for now and do a merge?